### PR TITLE
Update weapon.ts

### PR DESCRIPTION
**Pull request by EpicEfeathers**

### CODE REVIEW

**Comment by developomp** [comment]
Can you also fill out `WeaponName`?

File: libs/wb-types/src/weapon.ts [hunk 1]
```diff
@@ -42,17 +42,17 @@ export enum Weapon {
     SCAR = "p93",
     TacticalShotgun = "p94",
     VEK = "p95",
-    UNKNOWN_WEAPON_96 = "p96",
-    UNKNOWN_WEAPON_97 = "p97",
+    Desert = "p96",
+    Auto = "p97",
```
Comment (developomp) [inline libs/wb-types/src/weapon.ts:46]: The weapon name needs to be more specific than `Auto` and `Desert`

Comment (developomp) [reply libs/wb-types/src/weapon.ts:46]: https://discord.com/channels/989707899519524924/989707899519524927/1314345792764641280

They are what they're called in-game, so we're following that

**Comment by EpicEfeathers** [comment]
> Can you also fill out `WeaponName`?

I'll try doing this soon.

File: libs/wb-types/src/weapon.ts [hunk 2]
```diff
@@ -42,17 +42,24 @@ export enum Weapon {
...
     LMG = "p98",
-    UNKNOWN_WEAPON_103 = "p103",
-    UNKNOWN_WEAPON_104 = "p104",
-    UNKNOWN_WEAPON_105 = "p105",
-    UNKNOWN_WEAPON_101 = "p101",
-    UNKNOWN_WEAPON_110 = "p110",
+    UNRELEASED = "p99",
```
Comment (developomp) [inline libs/wb-types/src/weapon.ts:48]: replace `UNRELEASED` -> `UNRELEASED_WEAPON_99` for disambiguation with other unreleased weapons

**Comment by developomp** [comment]
Don't forget `SmokeGrenade = "p81"`

**Comment by EpicEfeathers** [comment]
Changed those two things. I will probably fill out `WeaponName` this week, unless you need it right now.

**Comment by developomp** [comment]
Nah, no rush. Take your time.

**Comment by EpicEfeathers** [comment]
Alright `WeaponName` should be complete. Change the names if you don't like them.

**Pull request merged**